src/utils/parser/utils.js

```diff
@@ -99,14 +99,23 @@ function getFunctionName(path) {
     return parent.id.name;
   }
 
+  if (parent.right && isFunction(parent.right)) {
+    return parent.left.property.name;
+  }
+
+  if (parent.right && isFunction(parent.right)) {
```

Comment:
This `if` condition is the same as the one above, it can be removed, otherwise this looks great.